### PR TITLE
WIP Feature/issuer api endpoints

### PR DESCRIPTION
**Pull request by morrieinmaas**
WIP for issuer endpoint.

### CODE REVIEW

**Comment by morrieinmaas** [comment]
Let's wait with actually reviewing it until we have michael's feedback on how to refactor and we can implement that here right away.

File: app/routers/issuer.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+            # connection_id = invite["connection_id"]
+            inviteURL = invite["invitation_url"]
+
+            qr = qrcode.QRCode(version=1, box_size=10, border=5)
+            qr.add_data(inviteURL)
+            qr.make(fit=True)
+            img = qr.make_image(fill="black", back_color="white")
+            buffer_img = io.BytesIO()
+            img.save(buffer_img, format="PNG")
+            await controller.terminate()
```
Comment (morrieinmaas) [inline app/routers/issuer.py:113]: we don't need to call terminate in a context manager

**Review by morrieinmaas — COMMENTED** [review]
Hey some nice effort here. There's a few things in the comments. Should be straight forward to fix. 

Also all the other cases where we now also want to replace awaiting things explicitly should be wrapped in a context manager just as a general remark. I'm happy to go tidy that up.

File: app/routers/issuer.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+            return StreamingResponse(resp_img, media_type="image/png")
+    except Exception as e:
+        err_trace = traceback.print_exc()
+        logger.error(
+            f"Failed to create qrcode. The following error occured:\n{e!r}\n{err_trace}"
+        )
+        raise e
+
+
+# TODO Decide where this endpoint to lie
```
[the remark this answers is not in the document]
Comment (vineeth14) [reply app/routers/issuer.py:124]: get-connection-id

[the remark this answers is not in the document]
Comment (morrieinmaas) [reply app/routers/issuer.py:116]: Hmm ah ok. And why is that? Just out of curiousity. FIne for now then I suppose

**Pull request merged**